src/lib/Serializer/Normalizer/Suggestion/ParentLocationCollectionNormalizer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Search\Serializer\Normalizer\Suggestion;
+
+use ArrayObject;
+use Ibexa\Contracts\Search\Model\Suggestion\ParentLocationCollection;
+use Symfony\Component\Serializer\Normalizer\CacheableSupportsMethodInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareTrait;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class ParentLocationCollectionNormalizer implements
+    NormalizerInterface,
+    NormalizerAwareInterface,
+    CacheableSupportsMethodInterface
+{
+    use NormalizerAwareTrait;
+
+    /**
+     * @param \Ibexa\Contracts\Search\Model\Suggestion\ParentLocationCollection $object
+     * @param array<string, mixed> $context
+     *
+     * @return ArrayObject<int,mixed>.
+     */
+    public function normalize($object, string $format = null, array $context = []): ArrayObject
+    {
+        $normalizedData = [];
+
+        foreach ($object as $parentLocation) {
+            $normalizedData[] = $this->normalizer->normalize($parentLocation);
+        }
+
+        return new ArrayObject($normalizedData);
+    }
+
+    public function supportsNormalization($data, string $format = null): bool
+    {
+        return $data instanceof ParentLocationCollection;
+    }
+
+    public function hasCacheableSupportsMethod(): bool
+    {
+        return __CLASS__ === static::class;
```

Comment:
Since this class is `final`, check is unnecessary.
```suggestion
        return true;
```